js/snippet.js

```diff
@@ -1,11 +1,13 @@
 try {
-	const statifyReq = new XMLHttpRequest();
-	statifyReq.open(
-		'GET',
-		document.getElementById('statify-js-snippet').getAttribute('data-home-url')
-		+ '?statify_referrer=' + encodeURIComponent(document.referrer)
-		+ '&statify_target=' + encodeURIComponent(location.pathname + location.search)
-	);
-	statifyReq.send(null);
-} catch (e) {
+	jQuery.ajax( {
```

Comment:
Can we rely on jQuery being loaded on every WP site? Just because WP loads jQuey by default, this does not mean that a theme may change this behavior, right?

Reply:
No, we shouldn't. Sure around 99% (guess) of all WP sites have jQuery active, but I for myself disabled it for non-admin users on multiple sites, so I wonder why I've used it here at all...

Reverted back to vanilla JS in the next commit.